src/components/AnnouncementList.jsx

```diff
@@ -0,0 +1,61 @@
+import React, { useState } from "react";
+import Announcement from "./Announcement";
+
+const announcements = [
+  {
+    date: "10 Nov 2022",
+    title: "New dev docs launched!",
+    info: `The new developer docs are now available in beta! The older site is still available in case you need to access legacy documentation.`,
+    link: "https://old-developers.helpshift.com",
+    linkTitle: "old-developers.helpshift.com",
+  },
+  {
+    date: "10 Oct 2022",
+    title: "SDK X 10.2.0 released",
+    info: `SDK X 10.2.0 is now released for iOS and Android! Upgrade to get the latest features`,
+  },
+  {
+    date: "1 June 2022",
+    title: "Feedback+ SDK released",
+    info: `Helpshift's first SDK with a free tier to help you get early feedback from your users`,
+    link: "/freesdk",
+    linkTitle: "Read the guide",
+  },
+];
+
+export default function AnnouncementList() {
+  const [expanded, setExpanded] = useState(false);
+  const _handleExpandedClick = () => {
+    setExpanded(!expanded);
+  };
+
+  if (!announcements.length) {
+    return null;
+  }
+
+  return (
+    <section
+      className="overflow-y-auto lg:pb-8 dark:bg-none dark:border dark:border-gray-700 dark:border-solid bg-white/40 dark:bg-white/0 backdrop-blur-sm px-4 rounded-xl"
+      style={{
+        maxHeight: "90vh",
+      }}
+    >
+      <div className="flex justify-between mt-4 mb-4 lg:mt-6 items-center">
+        <h2 className="text-lg md:text-2xl mb-0">Announcements</h2>
+        <button
+          onClick={_handleExpandedClick}
+          className="py-2 px-4 rounded-md text-xs bg-transparent text-gray-800 dark:text-gray-100 border border-solid border-gray-700 dark:border-gray-600 shadow cursor-pointer lg:hidden"
+        >
+          {!expanded ? "Show" : "Hide"}
```

Comment:
Assuming that we are not following the usual hs convention avoiding inlining JS/JSX.
Not a problem as such, but wondering about the reason.

Reply:
I can update this portion to follow that convention. There's no specific reason for having it this way (besides the fact that it was a tiny one-liner).

Reply:
Here it is convenient but for longer lines which span multiple lines while reading it is difficult.
So for convention we say don't inline.

Reply:
Would prefer not inlining here.

Reply:
Would be interesting to see how the general community here feels about this convention.
I think in most of the code bases inlining is followed.
Keeping the reading aspect on case to case basis can also work but review becomes a bit more complex then.

Reply:
If there is a resistance and substantial reasoning then we can change the rule in future.

Reply:
I'll remove the inlining and push a new commit for now to follow HS conventions

Reply:
Pushed a commit for this: 789d73d